chunked_pooling/mteb_chunked_eval.py

```diff
@@ -296,3 +300,9 @@ def _prune(queries, corpus, relevant_docs, prune_size):
                 new_corpus['test'][x] = corpus['test'][x]
             new_queries['test'][key] = queries['test'][key]
         return new_queries, new_corpus, new_relevant_docs
+
+    def _calculate_metrics_from_split(*args, **kwargs):
+        pass
+
+    def _evaluate_subset(*args, **kwargs):
```

Comment:
Where are `_calculate_metrics_from_split()` and `_evaluate_subset()` used?

Reply:
Or, what is the plan with them?

Reply:
In some newer MTEB version the AbsTask class requires them to be implemented but as we overwrite the evaluation method without using them they are useless however cause errors when they are not implemented by the non-abstract classes